sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -401,6 +395,36 @@ and
 - Added overloads to BlockBlobClient.getBlobOutputStream to allow users to provide parallel transfer options, http headers, metadata, access tier, and request conditions.
 
 
+## 11.1.1 (2019.04.30)
```

Comment:
Why did this section move up here?  Also, this appears to refer to track 1.

Reply:
Agreed, it looks like this change is not required.

Reply:
This was autogenerated, going to manually change it back. Thanks for pointing it out